chroma-manager/tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -16,28 +17,10 @@ def test_yum_update(self):
         )
         self.assertEqual(response.successful, True, response.text)
         hosts = response.json['objects']
-        self.assertEqual(len(hosts), len(self.TEST_SERVERS))
-
-        # Ensure that IML notices its storage servers needs upgraded
-        for host in hosts:
-            # wait for an upgrade available alert
-            self.wait_for_assert(lambda: self.assertHasAlert(host['resource_uri'],
-                                                             of_type='UpdatesAvailableAlert'))
-            alerts = self.get_list("/api/alert/", {'active': True,
-                                                   'alert_type': 'UpdatesAvailableAlert'})
-
-            # Should be the only alert
-            # FIXME HYD-2101 have to filter these alerts to avoid spurious ones.  Once that
-            # ticket is fixed, remove the filter so that we are checking that this really is
-            # the only alert systemwide as it should be.
-            alerts = [a for a in alerts if a['alert_item'] == host['resource_uri']]
-            self.assertEqual(len(alerts), 1)
-
-            # Should be an 'updates needed' alert
-            self.assertRegexpMatches(alerts[0]['message'], "Updates are ready.*")
-
-            # The needs_update flag should be set on the host
-            self.assertEqual(self.get_json_by_uri(host['resource_uri'])['needs_update'], True)
+        # TODO: one of the storage servers is a:
+        #       "profile": "posix_copytool_worker",
+        #       we need to exclude it programatically
+        self.assertEqual(len(hosts), len(self.TEST_SERVERS) - 1)
```

Comment:
api_testcase_with_test_reset.py : config_servers() gets this for you

Reply:
Done.  Thanks much for the suggestion!